packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -707,7 +707,10 @@ test(`zcfSeat from zcf.makeEmptySeatKit - only these properties exist`, async t
   const { zcf } = await setupZCFTest();
   const makeZCFSeat = () => zcf.makeEmptySeatKit().zcfSeat;
   const seat = makeZCFSeat();
-  t.deepEqual(getStringMethodNames(seat), expectedStringMethods.sort());
+  t.deepEqual(
+    getStringMethodNames(seat).filter(name => !name.startsWith('__')),
```

Comment:
If you create reusable testing helper for these, see if you can use it here too.

Reply:
It’s my preference to keep tests readable at the expense of repetition. In this case, I believe the code is offering an explanation.